docs/Files/UI/Solution-Manager/Library/Calendars/Calendars.md

```diff
@@ -0,0 +1,17 @@
+---
+lang: en-us
+title: Managing Calendars
+viewport: width=device-width, initial-scale=1.0
+---
+
```

Comment:
this file should be called Managing-Calendars.md

Reply:
Changed to use Managing-Calendars.md